ambari/src/main/java/io/brooklyn/ambari/AmbariClusterImpl.java

```diff
@@ -179,6 +190,11 @@ public void start(Collection<? extends Location> locations) {
         return Entities.descendants(this, AmbariServer.class);
     }
 
+    @Override
+    public List<String> getExtraStackDefinitionsUrls() {
+        return this != null ? getConfig(STACK_DEFINITION_URLS) : ImmutableList.<String>of();
```

Comment:
Shouldn't be more like `return getConfig(STACK_DEFINITION_URLS);` ?

`this` is always different than `null` as you call this method, and `getConfig(STACK_DEFINITION_URLS)` is set with a default empty list.